sklearn/multioutput.py

```diff
@@ -233,6 +233,18 @@ class MultiOutputRegressor(RegressorMixin, _MultiOutputEstimator):
     ----------
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_regression
+    >>> from sklearn.multioutput import MultiOutputRegressor
+    >>> from sklearn.linear_model import LinearRegression
```

Comment:
Could you use a RidgeRegressor instead of a LinearRegression. Usually, we don't encourage to not apply any regularization so I would prefer to not add this in an example.

Reply:
Makes sense. Done.